OrbitGl/TimeGraph.cpp

```diff
@@ -211,6 +211,25 @@ void TimeGraph::PanTime(int a_InitialX, int a_CurrentX, int a_Width,
   NeedsUpdate();
 }
 
+void TimeGraph::VerticallyScrollIntoView(const TextBox* text_box) {
+  CHECK(text_box != nullptr);
+  if (text_box->GetPos() == Vec2::Zero())  // Have to update the information of
```

Comment:
Please use the following style for this:

```
// Have to update the information of this TextBox before align
  if (text_box->GetPos() == Vec2::Zero()) {
    NeedsUpdate();
}
```

Reply:
Actually, see my next comment. This condition and call to NeedsUpdate() have no effect.